hie-plugin-api/Haskell/Ide/Engine/PluginUtils.hs

```diff
@@ -9,10 +9,16 @@ module Haskell.Ide.Engine.PluginUtils
   , mapEithers
   , missingParameter
   , incorrectParameter
+  , validatePlugins
```

Comment:
I do not believe this funcionality belongs in `hie-plugin-api`, it will only be used when a plugin is integrated, so belongs either in `Haskell.Ide.Engine.Dispatcher` or a `Haskell.Ide.Engine.Utils` file in the same place.